pandas/tests/plotting/test_frame.py

```diff
@@ -2995,6 +2995,23 @@ def test_secondary_axis_font_size(self, method):
         self._check_ticks_props(axes=ax.right_ax,
                                 ylabelsize=fontsize)
 
+    @pytest.mark.slow
+    def test_x_string_values_ticks(self):
+        # GH: 7612
```

Comment:
can you add a 1-liner explaining what this test is doing; all of the issue references can be on a single line

Reply:
I added the following one-liner:
# Test if string plot index have a fixed xtick position
# Test if multiindex plot index have a fixed xtick position